### ISSUE TITLE
EOS used in the Refprop Worksheet

### ISSUE THREAD

**Issue by kurzeos** [usnistgov/REFPROP-wrappers#609]
Refprop 10.0 comes with an .xls work sheet REFPROP_work.xls. What is the default Equation of state in this excel sheet (GERG?), and how can I check? If its not GERG, how can I set GERG as default?
Thx

**Comment by ianhbell** [comment]
No, the default are the more-accurate pure fluid EOS and the more accurate (most of the time) mixture models. See the VBA code for how to enable GERG mode in the Excel sheet.

**Comment by svitale1** [comment]
I must use Robinson in excel. I need to also change some of the units of the property units being transferred from REFPROP to excel . Please help. Svitale@ptd.net

**Comment by svitale1** [comment]
Matthew,I’ve spent the last 7 days doing just that.  I do not know VBA code and after crashing excel and needing to reinstall REFPROP several times, I am lost. I would pay for help if I knew where to go or who to contact.I did not mean to be rude by reposting my issue.  For me it is still an issue!Sincerely,SteveSent from my iPhoneOn Jul 18, 2025, at 11:19 AM, Matthew Boyd ***@***.***> wrote:﻿Matthew-Boyd left a comment (usnistgov/REFPROP-wrappers#609)
@svitale1 I refer you to the REFPROP.XLS workbook packaged with REFPROP, specifically the section Options Available in the VBA Code to Change how REFPROP Works in the Welcome worksheet. For units, refer to the Units and/or the Units Conversions worksheets.
If that doesn't help, please create a new issue in this repo with more context. Closing this issue as resolved.

**Comment by Matthew-Boyd** [comment]
Wonderful, happy to help. Thanks for the message.
